test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,33 @@
+const axios = require('axios');
+const { expect } = require('chai');
+
+const urlBase = 'https://api.github.com/user/following/aperdomob';
+
+const keyPath = axios.create({
+  path: urlBase,
+  headers: { Authorization: `token ${process.env.ACCESS_TOKEN}` }
+});
+
+describe('Stage 9 - PUT: Following', () => {
+  let follow;
+  before(async () => {
+    follow = await keyPath.put(urlBase);
+  });
+
+  it('2. Follow user', async () => {
+    expect(follow.status).to.equal(204);
+    expect(follow.data).to.be.eql('');
+  });
+
+  it('3. Follow check', async () => {
+    const check = await keyPath.get(urlBase);
```

Comment:
in this case the get should be done over: `https://api.github.com/user/following`

Reply:
I have a question, [in this section](https://docs.github.com/es/rest/users/followers#check-if-a-person-is-followed-by-the-authenticated-user) i read that if i send a GET to .../user/following/aperdomob and i receive a 204 in status, is enough confirmation that I'm following that user... my question is: the method that you are suggesting, is for "educational" purpose and should be done as you request, or we can use the Github method?

Reply:
No, please note that the URIs are different.

To follow the user is PUT method to URI: /user/following/aperdomob
To get the users you are following, is GET method to URI: /user/following/

Reply:
Thanks for the explanation. An additional comment... i tried a lot to pass the check after add .find and after many tries, i found that if you follow a lot of people (like me) the /following has something like pagination... so, in the first .find that test made, never found 'aperdomob'... i decided to unfollow a lot of people and then, the check passed :)

Reply:
hehe you could use pagination sending query params to increase the number of registers retrieved by the API, you will see how to do it in future sections.